src/index.ts

```diff
@@ -6,14 +6,14 @@ export * from './page-view-tracker';
 export * from './telemetry-client';
 export * from './track-event-binding-behavior';
 
-import {FrameworkConfiguration, LogManager} from 'aurelia-framework';
+import {FrameworkConfiguration, LogManager, PLATFORM} from 'aurelia-framework';
 import {ConfigurationBuilder, ConfigurationBuilderImpl} from './configuration';
 import {LogAppender} from './log-appender';
 import {GlobalErrorTracker} from './global-error-tracker';
 import {PageViewTracker} from './page-view-tracker';
 
 export function configure(aurelia: FrameworkConfiguration, callback?: ((c: ConfigurationBuilder) => void)) {
-  aurelia.globalResources(['./track-event-binding-behavior']);
+  aurelia.globalResources(PLATFORM.moduleName['./track-event-binding-behavior']);
```

Comment:
I think it should be:

```ts
aurelia.globalResources([PLATFORM.moduleName('./track-event-binding-behavior')]);
```

as `PLATFORM.moduleName` is a function.

Reply:
Yes you are right, 
Sorry I changed some other things, too and tried to submit a clean patch but that was the wrong version.
I pushed a fixed.